integration_tests/Invoker.php

```diff
@@ -0,0 +1,116 @@
+<?php
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+declare(strict_types=1);
+
+namespace Google\Generator\IntegrationTests;
+
+class Invoker
+{
+    public static function Invoke(string $protoPath)
```

Comment:
Do I want to know why this is `Invoke` but `SourceComparer` has a `compare` method (lower case)?

Reply:
Oops! I'll fix the casing, just my mistake ;)